metadata.rb

```diff
@@ -5,7 +5,7 @@
 description      'Installs and Configures Supermarket from the Omnibus packages on packages.chef.io'
 source_url       'https://github.com/chef-cookbooks/supermarket-omnibus-cookbook'
 issues_url       'https://github.com/chef-cookbooks/supermarket-omnibus-cookbook/issues'
-chef_version     '>= 12.5' if respond_to?(:chef_version)
+chef_version     '>= 12.5'
```

Comment:
This should get bumped to 12.9 which will take any old metadata item @robbkidd

Reply:
Do you think I'd need to make any other changes? Bump the major version because the required minimum Chef has changed (even if _that_ didn't bump the major)?